src/components/nodes/boundaryEvent/boundaryEvent.vue

```diff
@@ -50,50 +50,28 @@ export default {
         .filter(model => model.component)
         .find(model => taskIds.includes(model.component.node.type));
     },
-    updateBoundaryShape(dashLength) {
+    setShapeStrokeDasharry(dashLength) {
       this.shape.attr({
         body: {
-          'strokeDasharray': dashLength,
+          strokeDasharray: dashLength,
         },
         body2: {
-          'strokeDasharray': dashLength,
+          strokeDasharray: dashLength,
         },
       });
     },
-    renderInterrupting(isCancelActivity) {
+    toggleInterruptingStyle(isCancelActivity) {
       const dashedLine = 5;
       const solidLine = 0;
-      isCancelActivity ? this.updateBoundaryShape(solidLine) : this.updateBoundaryShape(dashedLine);
+      this.setShapeStrokeDasharry(isCancelActivity ? solidLine : dashedLine);
     },
   },
   mounted() {
-    // Now, let's add a rounded rect to the graph
     this.shape = new EventShape();
-    let bounds = this.node.diagram.bounds;
-    this.shape.position(bounds.get('x'), bounds.get('y'));
-    this.shape.resize(bounds.get('width'), bounds.get('height'));
-    this.shape.attr({
-      body: {
-        stroke: '#212529',
-        strokeWidth: 0.85,
-        fill: '#FFF',
-      },
-      body2: {
-        stroke: '#212529',
-        fill: '#FFF',
-        strokeWidth: 0.85,
-      },
-      label: {
-        text: this.node.definition.get('name'),
-        refY: '130%',
-      },
-      image: {
-        'ref-x': 5,
-        'ref-y': 5,
-        'width': bounds.get('width') - 10,
-        'height': bounds.get('height') - 10,
-      },
-    });
+    const { x, y, width, height } = this.node.diagram.bounds;
+    this.shape.position(x, y);
+    this.shape.resize(width, height);
+    this.shape.attr('label/text', this.node.definition.get('name'));
```

Comment:
Could this all be extracted to a new method? That would make the intention clearer IMO.